code/__HELPERS/maths.dm

```diff
@@ -17,8 +17,8 @@ var/list/sqrtTable = list(1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4,
 	var/a = arccos(x / sqrt(x*x + y*y))
 	return y >= 0 ? a : -a
 
-/proc/Ceiling(x)
-	return -round(-x)
+/proc/Ceiling(x, y)
+	return -round(-x / y) * y

```

Comment:
y defaults to 0, and it divides by y. This is a division by 0 runtime waiting to happen.